### PR TITLE
Added share session feature

### PR DESCRIPTION
**Pull request by konifar**
DroidKaigi web page added session detail page :+1: 
https://droidkaigi.github.io/2016/timetable.html#session-2016-02-18-11-30-12-00-a

So I added the feature that user can share url from session detail page.

untitled

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched/model/Session.java [hunk 1]
```diff
@@ -65,6 +64,16 @@
     @SerializedName("slide_url")
     public String slideUrl;
 
+    @Column(defaultExpr = "''")
+    @Nullable
+    @SerializedName("movie_url")
+    public String movieUrl;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/model/Session.java]: We can add youtube url or ustream url.

File: app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java [hunk 2]
```diff
@@ -14,4 +18,11 @@ public static Intent toBrowser(String url) {
         return new Intent(Intent.ACTION_VIEW, uri);
     }
 
+    public static void share(Context context, @NonNull String url) {
+        Intent intent = new Intent(Intent.ACTION_SEND);
+        intent.setType("text/plain");
+        intent.putExtra(Intent.EXTRA_TEXT, url);
+        context.startActivity(Intent.createChooser(intent, context.getString(R.string.share)));
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java]: If there is more good way, please tell me :bow:

Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java]: Guess we can use ShareActionProvider but not sure which is better.

[Adding an Easy Share Action | Android Developers](http://developer.android.com/intl/ja/training/sharing/shareaction.html)

Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java]: Shouldn't we add a hash tag of twittter? :bird:

**Comment by hotchemi** [comment]
Good work!

File: app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java (hunk 2 above)
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java]: I didn't know `ShareActionProvider`! Let me check it!

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java]: Sounds good to add a hash tag :+1: 
Which is better `#droidkaigi #droidkaigiA`  or `droidkaigi`?

Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java]: Guess #droidkaigi is enough, but any thoughts? @hkurokawa @KeithYokoma

**Pull request merged**